isok-broker/src/transport/kafka.rs

```diff
@@ -0,0 +1,128 @@
+use crate::config::KafkaConfig;
+use crate::transport::{ResultTransport, TransportError};
+use isok_data::broker_rpc::CheckResult;
+use prost::Message;
+use rdkafka::producer::{FutureProducer, FutureRecord};
+use rdkafka::ClientConfig;
+use std::time::Duration;
+
+pub struct KafkaMessageBroker {
+    producer: FutureProducer,
+    topic: String,
+}
+
+impl ResultTransport for KafkaMessageBroker {
+    async fn process_result(&self, result: &CheckResult) -> Result<(), TransportError> {
+        let mut buffer = Vec::new();
+        result.encode(&mut buffer).unwrap();
+        let record = FutureRecord::to(&self.topic)
+            .payload(&buffer)
+            .key(&result.id_ulid);
+
+        self.producer
+            .send(record, Duration::from_secs(2))
```

Comment:
We could declare this value in a static for the whole module for easing discovery an fine tuning

```rust
pub static QUEUE_TIMEOUT: Duration = Duration::from_secs(2);
```

Reply:
Applied